website/vercel.json

```diff
@@ -747,116 +747,124 @@
       "destination": "https://hub.cloudquery.io/plugins/destination/cloudquery/sqlite"
     },
     {
-      "source": "/integrations/file",
```

Comment:
Changes here sort the integrations redirects and fix redirects that have both source and destinations such as `/integrations/file/mysql`